sdk/core/core/test/cmocka/test_span.c

```diff
@@ -20,6 +21,52 @@
   assert_true(az_span_append_uint8(buffer, 'a', NULL) == AZ_ERROR_ARG);
 } */
 
+void az_single_char_ascii_lower_test()
+{
+  for (uint8_t i = 0; i <= SCHAR_MAX; i++)
```

Comment:
nit: prefix ++

Reply:
I will have to adjust to this, and unlearn this habit :p